config.yaml

```diff
@@ -3,6 +3,9 @@ options:
     type: string
     default: "192.168.251.5/24"
     description: gNodeB IP Address
+  gnb-interface:
+    type: string
+    description: Host interface to use for the RAN Network.
```

Comment:
We many want to explicitly say what happens if this is not specified.

Reply:
Added a quick description